README.md

```diff
@@ -294,3 +294,38 @@ Accessing the command built using LINQ
 ```c#
 var command = ((ClientQueryProvider<Category>) query.Provider).Command;
 ```
+
+## Custom Services
```

Comment:
Question: why does Stores need to be handled differently than other endpoints?

Reply:
it's just for example as it's not currently supported, it will be handled soon like other endpoints